apps/chat/src/components/Chat/Chat.tsx

```diff
@@ -738,69 +732,72 @@ export const ChatView = memo(() => {
-                    {mergedMessages?.length > 0 && (
-                      <div className="flex flex-col" data-qa="chat-messages">
-                        {mergedMessages.map(
-                          (
-                            mergedStr: [Conversation, Message, number][],
-                            i: number,
-                          ) => (
-                            <div
-                              key={i}
-                              className="flex w-full"
-                              data-qa={
-                                isCompareMode
-                                  ? 'compare-message-row'
-                                  : 'message-row'
-                              }
-                            >
-                              {mergedStr.map(
-                                ([conv, message, index]: [
-                                  Conversation,
-                                  Message,
-                                  number,
-                                ]) => (
-                                  <div
-                                    key={conv.id}
-                                    className={classNames(
-                                      isCompareMode &&
-                                        selectedConversations.length > 1
-                                        ? 'w-[50%]'
-                                        : 'w-full',
-                                    )}
-                                  >
-                                    <div className="size-full">
-                                      <MemoizedChatMessage
-                                        key={conv.id}
-                                        message={message}
-                                        messageIndex={index}
-                                        conversation={conv}
-                                        isLikesEnabled={enabledFeatures.has(
-                                          Feature.Likes,
-                                        )}
-                                        editDisabled={!!notAllowedType}
-                                        onEdit={onEditMessage}
-                                        onLike={onLikeHandler(index, conv)}
-                                        onDelete={() => {
-                                          handleDeleteMessage(index, conv);
-                                        }}
-                                        onRegenerate={
-                                          index === mergedMessages.length - 1 &&
-                                          showLastMessageRegenerate
-                                            ? onRegenerateMessage
-                                            : undefined
-                                        }
-                                        messagesLength={mergedMessages.length}
-                                      />
+                    <div ref={chatMessagesRef}>
```

Comment:
Do we need an additional div? Maybe we can put this ref in the div below?

Reply:
No, we can't - there is condition `mergedMessages?.length > 0`